feature-set/src/lib.rs

```diff
@@ -1116,6 +1116,10 @@ pub mod reenable_zk_elgamal_proof_program {
     solana_pubkey::declare_id!("zkemPXcuM3G4wpMDZ36Cpw34EjUpvm1nuioiSGbGZPR");
 }
 
+pub mod disable_all2all_tests {
+    solana_pubkey::declare_id!("9PsiyXopc2M9DMEmsEeafNHHHAUmPKe9mHYgrk6fHPyx");
```

Comment:
we are not breaking consensus for a toy. use `ClusterType`

Reply:
Discussed on zoom: Need to implement an account state check rather than an actual feature gate. The account state check should be possible when a Bank instance is available.